platform/ios/src/MGLMapView.mm

```diff
@@ -397,7 +398,8 @@ - (void)commonInit
 
     // setup mbgl map
     mbgl::DefaultFileSource *mbglFileSource = [MGLOfflineStorage sharedOfflineStorage].mbglFileSource;
-    _mbglMap = new mbgl::Map(*_mbglView, *mbglFileSource, mbgl::MapMode::Continuous, mbgl::GLContextMode::Unique, mbgl::ConstrainMode::None, mbgl::ViewportMode::Default);
+    mbgl::ThreadPool workerThreadPool(4);
```

Comment:
Fixed this - is the stack allocation okay in `bin/render.cpp`, `bin/glfw.cpp` and the tests?